src/shared/Microsoft.Git.CredentialManager/Settings.cs

```diff
@@ -392,6 +398,30 @@ public bool IsCertificateVerificationEnabled
             }
         }
 
+        public string SslCaInfo
+        {
+            get
+            {
+                // First, verify that, if schannel is the HTTP backend, sslCaInfo must be explicitly enabled to be used
+                if (TryGetSetting(null, KnownGitCfg.Http.SectionName, KnownGitCfg.Http.SslBackend, out string backend) &&
+                    StringComparer.OrdinalIgnoreCase.Equals(backend, "schannel") &&
+                    (!TryGetSetting(null, KnownGitCfg.Http.SectionName, KnownGitCfg.Http.SchannelUseSslCaInfo, out string schannelUseSslCaInfo) ||
```

Comment:
Is there any chance that we can call `git config --type=path http.sslCAInfo`? The [`--type=path` option](https://git-scm.com/docs/git-config#Documentation/git-config.txt---typelttypegt) would ensure that we benefit from Git's own path interpolation.

Reply:
This would involve some changes to [GitConfiguration.cs](https://github.com/microsoft/Git-Credential-Manager-Core/blob/main/src/shared/Microsoft.Git.CredentialManager/GitConfiguration.cs) (like adding an optional `isPath` argument to `TryGet(...)` or creating a `TryGetPath(...)` method), but I really like the benefits of using Git's path interpolation logic (and not replicating it here). @mjcheetham what do you think?

Does this handle the environment variable version of the config as well (in this case, `GIT_SSL_CAINFO`)? Or is `%(prefix)` (and any other "magic" path interpolation) not supported with environment variables?

Reply:
This does _not_ handle the environment variables (but neither does Git interpolate prefix-relative paths in environment variables, so I think we're good).

Reply:
> @mjcheetham what do you think?

I think this should be discussed in a separate PR/issue, so let's leave this as is today for this change.